orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -808,16 +808,3 @@ def upsample_nearest2d(g, input, output_size, scale_factors):
 @register_symbolic("upsample_nearest3d")
 def upsample_nearest3d(g, input, output_size, scale_factors):
     return _upsample_nearest(g, input, output_size, scale_factors, "upsample_nearest3d")
-
-
-@register_symbolic("upsample_bilinear2d")
-def upsample_bilinear2d(g, input, output_size, align_corners, scale_factors):
-    return g.op(
-        "org.pytorch.aten::ATen",
```

Comment:
after removing the aten fallback and switch using Resize as fw, and ResizeGrad as bw, do we lose some kernel converge for following cases: 

"The gradient kernel currently only supports the bilinear mode. The gradient kernel does not currently support antialiasing."